digitalfabrik/entitlementcard
# 1623: Add i18n for administration

### Short description
Move labels from objects in the database to translation.json

### Proposed changes
- [x] Added i18n to administration
- [x] Use labels from translation file instead of database/object-json
- [x] remove adding labels before saving new application to the database

### Side effects
- May break everything especially things related to applications

### Testing
- Open application for `/beantragen` and check if all labels for all cases are displayed properly
- create applications for different type, e.g. blue entitlement card for work at organization, julica, ... and view if all labels are correctly displayed in the digitale druckerei "Eingehende Anträge" view.
- check if older applications created without these changes here are still displayed properly
- check if sent emails still look as expected

### Todo for follow up ticket
- replace all german strings with labels from translation file

### Resolved issues
Fixes: #1623
